app/views/partials/nav.erb

```diff
@@ -1,11 +1,11 @@
 <ul class="nav nav-pills custom-nav">
-  <li><a href='/'>Home</a></li>
+  <li id='home'><a href='/'>Home</a></li>
   <li role="presentation" class="dropdown custom-dropdown">
     <a class="dropdown-toggle" data-toggle="dropdown" href="#" role="button" aria-haspopup="true" aria-expanded="false">
       Stations <span class="caret"></span>
     </a>
     <ul class="dropdown-menu">
-      <li><a href='/stations'>Stations</a></li>
+      <li id="stations_button"><a href='/stations'>Station Index</a></li>
```

Comment:
It's more descriptive and differentiates itself from the title of the dropdown menu.